specification/managementgroups/resource-manager/Microsoft.Management/preview/2024-01-01-preview/management.json

```diff
@@ -0,0 +1,2383 @@
+  },
+  "schemes": [
+    "https"
+  ],
+  "consumes": [
+    "application/json"
+  ],
+  "produces": [
+    "application/json"
+  ],
+  "security": [
+    {
+      "azure_auth": [
+        "user_impersonation"
+      ]
+    }
+  ],
+  "securityDefinitions": {
+    "azure_auth": {
+      "type": "oauth2",
+      "authorizationUrl": "https://login.microsoftonline.com/common/oauth2/authorize",
+      "flow": "implicit",
+      "description": "Azure Active Directory OAuth2 Flow.",
+      "scopes": {
+        "user_impersonation": "impersonate your user account"
+      }
+    }
+  },
+  "tags": [
+    {
+      "name": "ManagementGroups",
+      "description": "A Management Group is a customer defined scope (grouping mechanism) that \nprovides access control (authorization), policy management and reporting. \nManagement Groups are organized in a strictly tree-based hierarchy.\n"
+    },
+    {
+      "name": "Operations",
+      "description": "Management operations supported by the Microsoft.Management resource provider.\n"
+    },
+    {
+      "name": "Entities",
+      "description": "A list of entities that belong to the Management Groups.\n"
+    },
+    {
+      "name": "Applications",
+      "description": "An Application is a customer created scope, where multiple subscriptions or applications can be grouped by. \n"
+    }
+  ],
+  "paths": {
+    "/providers/Microsoft.Management/managementGroups": {
+      "get": {
+        "tags": [
+          "ManagementGroups"
+        ],
+        "operationId": "ManagementGroups_List",
+        "description": "List management groups for the authenticated user.\n",
+        "parameters": [
+          {
+            "$ref": "#/parameters/ApiVersionParameter"
+          },
+          {
+            "$ref": "#/parameters/CacheControlHeader"
+          },
+          {
+            "$ref": "#/parameters/SkipTokenParameter"
+          }
+        ],
+        "responses": {
+          "200": {
+            "description": "OK",
+            "schema": {
+              "$ref": "#/definitions/ManagementGroupListResult"
+            }
+          },
+          "default": {
+            "description": "Error",
+            "schema": {
+              "$ref": "#/definitions/ErrorResponse"
+            }
+          }
+        },
+        "x-ms-pageable": {
+          "nextLinkName": "@nextLink"
+        },
+        "x-ms-examples": {
+          "ListManagementGroups": {
+            "$ref": "./examples/ListManagementGroups.json"
+          }
+        }
+      }
+    },
+    "/providers/Microsoft.Management/managementGroups/{groupId}": {
+      "get": {
+        "tags": [
+          "ManagementGroups"
+        ],
+        "operationId": "ManagementGroups_Get",
+        "description": "Get the details of the management group.\n",
+        "parameters": [
+          {
+            "$ref": "#/parameters/GroupIdParameter"
```

Comment:
>parameters/GroupIdParameter [](http://example.com/codeflow?start=23&length=27)

Get this from v5 common types ManagementGroupName definition (applies to all instances in this file).